turbinia/config/__init__.py

```diff
@@ -29,9 +29,9 @@
 # Look in homedir first, then /etc/turbinia, and finally in the source
 # config dir for config files
 CONFIGPATH = [
-    os.path.expanduser('~'),
-    '/etc/turbinia',
-    os.path.dirname(os.path.abspath(__file__))]
+    os.path.expanduser('~'), '/etc/turbinia',
+    os.path.dirname(os.path.abspath(__file__))
+]
```

Comment:
Hrm... I like having the close bracket on a new line, but then I'm not sure why it's putting multiple entries on the same line.  I prefer either/or.